src/sass/modules/_m-form-elements.scss

```diff
@@ -55,7 +55,7 @@ button.form-button-disabled {
     display: block;
     float: left;
     margin-left: -1.7em;
-    margin-top: 0.14em;
+    margin-top: 0.25em;
```

Comment:
Oh weird. I guess it's the `label::before` that's the button image. Makes sense. I was reading that as just the label itself which, now that I think about it, is silly.

Thanks for the clarification!